RubberduckBaseProject.csproj

```diff
@@ -45,7 +45,15 @@
                         AND '$(AssemblyName)' != 'RubberduckTestsCodeAnalysis'">
         <Analyzer Include="$(SolutionDir)\RubberduckCodeAnalysis\bin\RubberduckCodeAnalysis.dll" />
   </ItemGroup>
-
+  <!-- Commented out due to large number of false positives. Hopefully the package
+	   will get annotations, which would enable us to exclude those false positives.
+	   Ref: https://github.com/DotNetAnalyzers/IDisposableAnalyzers/issues/126
+  <ItemGroup>
+    <PackageReference Include="IDisposableAnalyzers">
+	  <Version>2.0.5</Version>
+	</PackageReference>
```

Comment:
Can that be just an `Analyzer` reference or does the `PackageReference` automatically add the analyzer?

Reply:
Adding the `PackageReference` automatically adds its to the analyzer somehow. I would have liked to add to the `Analyzer` to be more explicit but not sure I can do that without making it go weird.